pysparkling/tests/test_multiprocessing.py

```diff
@@ -155,15 +158,30 @@ def runtime(self, n=10, processes=1):
                                      serializer=cloudpickle.dumps,
                                      deserializer=pickle.loads)
             rdd = sc.parallelize(range(n), 10)
-            rdd.map(lambda _: time.sleep(0.1)).collect()
+            rdd.map(lambda _: time.sleep(0.01)).collect()
         return time.time() - start
 
-    @unittest.skipIf(platform.python_implementation() == 'PyPy',
-                     'test fails in PyPy')
+    @unittest.skipIf(platform.python_implementation() == 'PyPy', 'test fails in PyPy')
     def test_basic(self):
         t1 = self.runtime(processes=1)
         t10 = self.runtime(processes=10)
-        self.assertLess(t10, t1 / 2.0)
+
+        # Timings on my Windows computer:
```

Comment:
Could we add slightly more information on the computer? (namely # of core / frequency)

Reply:
Added